mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -68,18 +61,25 @@ static const float kModelMatrix[] = {0.83704215,  -0.36174262, 0.41049102, 0.0,
 //   MODEL_MATRICES (TimedModelMatrixProtoList, optional):
 //     If provided, will set the model matrices for the objects to be rendered
 //     during future rendering calls.
+//   GIF_TEXTURE (AssetTextureFormat, required):
+//     Will load a dynamic texture that is overlayed upon the GIF_ASSET .obj file
 //
 // Input side packets:
-//   TEXTURE (ImageFrame on Android / GpuBuffer on iOS, required):
-//     Texture to use with animation file.
-//   ANIMATION_ASSET (String, required):
-//     Path of animation file to load and render. Should be generated by
-//     //java/com/google/android/apps/motionstills/SimpleObjEncryptor with
-//     --compressed_mode=true.  See comments and documentation there for more
-//     information on custom .obj.uuu file format.
+//   ROBOT_TEXTURE (ImageFrame on Android / GpuBuffer on iOS, required):
+//     Texture to use for the Robot animation
+//   ROBOT_ASSET (String, required):
+//     Path of animation file to load and render for the Robot animation. Should
+//     be generated by SimpleObjEncryptor.jar with --compressed_mode=true.
+//   DINO_TEXTURE (ImageFrame on Android / GpuBuffer on iOS, required):
+//     Texture to use with the Dino animation file.
+//   DINO_ASSET (String, required):
+//     Path of animation file to load and render for the Dino animation. Should
+//     be generated by SimpleObjEncryptor.jar with --compressed_mode=true.
+//   GIF_ASSET (String, required):
```

Comment:
I currently have the GIF obj file as a single unmoving [picture frame](https://www.turbosquid.com/3d-models/free-obj-model-picture-frame/714537) upon which the GIF texture is loaded (changing each frame). The texturing works in the same way as every other object, so we could have some dynamic .obj animation sequence and do something interesting with the GIF (maybe a picture frame popping in and out, or warping edges). This would just be a matter of creating the actual object files in the same way Dino/Robot was done.